pkgs/data/icons/vimix-cursor-theme/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, lib
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "vimix-cursor-theme";
+  version = "2020-02-24";
+
+  src = fetchFromGitHub {
+    owner = "vinceliuice";
+    repo = "Vimix-cursors";
+    rev = version;
+    sha256 = "sha256-TfcDer85+UOtDMJVZJQr81dDy4ekjYgEvH1RE1IHMi4=";
+  };
+
+  installPhase = ''
+    mkdir -p            $out/share/icons/Vimix-cursors{,-white}
+
+    cp -r dist/*        $out/share/icons/Vimix-cursors
+    cp -r dist-white/*  $out/share/icons/Vimix-cursors-white
+  '';
+
+  meta = with lib; {
+    description = "An x-cursor theme inspired by Materia design and based on capitaine-cursors.";
```

Comment:
```suggestion
    description = "An x-cursor theme inspired by Material design and based on capitaine-cursors";
```

Reply:
This is actually the intended spelling, as far as I can tell: https://github.com/PapirusDevelopmentTeam/materia-kde

Reply:
seems like a typo based on https://github.com/vinceliuice/vimix-gtk-themes#readme
either way, period at the end should be removed

Reply:
Done :+1: